pkg/model/api/v1/secret/authorization.go

```diff
@@ -91,6 +91,9 @@ func (a *Authorization) GetCredentials() (string, error) {
 }
 
 func (a *Authorization) validate() error {
+	if a == nil {
```

Comment:
Excepting if you face an issue, I don't see why it would be necessary. If `Authorization` is not set, then yaml or json shouldn't call it.

Reply:
it was setting a default value for the token type

Reply:
checking this again, is not needed.